packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -15,19 +15,45 @@ import {
   TX_BATCH_SUBMITTER_LOG_TAG,
 } from '..'
 
+const environment = process.env.NODE_ENV
+const release = `batch-submitter@${process.env.npm_package_version}`
+
 /* Logger */
 const name = 'oe:batch_submitter:init'
-const log = new Logger({
-  name,
-  sentryOptions: {
-    release: `batch-submitter@${process.env.npm_package_version}`,
-    dsn: process.env.SENTRY_DSN,
-    tracesSampleRate: 0.05,
-  },
-})
+let log
+switch (environment) {
+  case 'kovan':
+  case 'mainnet':
+    log = new Logger({
+      name,
+      sentryOptions: {
+        release,
+        dsn: process.env.SENTRY_DSN,
+        tracesSampleRate: 0.05,
+        environment,
+      },
+    })
+    break
+  case 'goerli':
+    log = new Logger({
+      name,
+      sentryOptions: {
+        release,
+        dsn: process.env.SENTRY_DSN,
+        tracesSampleRate: 1, // Use 100% trace rate
```

Comment:
What do you think about making the `tracesSampleRate` configurable itself instead of coupling it to an environment? Now if we want sentry on a different testnet, then we have to add code here

Reply:
Also `NODE_ENV` is used in the JS ecosystem to be `production` or `development` to differentiate between those things. https://expressjs.com/en/advanced/best-practice-performance.html#set-node_env-to-production is an example. We have multiple "production" environments, in particular kovan and mainnet

Reply:
i'm open to it! i don't love the extra moving piece, but if trace rate is the only thing we will distinguish between goerli and the others happy to make this an extra config. what else might we want to be different?

Reply:
The only other thing that appears to be different is the `environment` field passed into `Metrics.labels`. That could also be configured independently.

Reply:
made this its own env var!